dev/run.sh

```
@@ -0,0 +1,246 @@
+
+echo -n "$prefix updating some Entr'Ouvert services folders user:group via chown..."
+chown authentic-multitenant:authentic-multitenant /var/lib/authentic2-multitenant/tenants -R &&
+chown hobo:hobo /var/lib/hobo/tenants -R &&
+chown chrono:chrono /var/lib/chrono/tenants -R &&
+chown combo:combo /var/lib/combo/tenants -R &&
+chown passerelle:passerelle /var/lib/passerelle/tenants -R &&
+chown wcs:wcs /var/lib/wcs -R &&
+echo " done! ✅"
+
+echo -n "$prefix verifying uploads & attachments permission folders..."
+[ -d /var/lib/wcs/tenants/*/attachments ] && (chown -R wcs:wcs /var/lib/wcs/tenants/*/attachments/ && echo -n " attachments done! ✅..." || echo -n " attachments failed! ❌...") || echo -n " attachments skipped! 🚫... "
```

Shellcheck (0.11.0)

[error] -d doesn't work with globs. Use a for loop.

(SC2144)